server/db/models/User.js

```diff
@@ -57,4 +58,25 @@ const User = db.define('user', {
   }
 })
 
+User.getStudentsByCohort = async function(cohortId) {
+  const cohortStudents = await CohortUser.findAll({
```

Comment:
So I think the following is equivalent and probably a bit faster since there's no for loop?  

User.findAll({
    include: [
      {
        attributes: [],
        model: CohortUser,
        where: {
          cohortId
        }
      }
    ]
  })
    .then(students => res.json(students))
    .catch(next)

Reply:
I guess my understanding of eager loading is off? Shouldn't the above code return an associated CohortUser record for each user? Like this (pseudo):

{ user: 'moe', id: 1, cohortUsers: [ { cohortUser: 'a', userId: 1 } ]}

But it looks like above you're using it to just create an inner join?

Reply:
Does it make more sense to set up the association differently? To me it makes more sense to establish the many-to-many relationship with something like:

User.belongsToMany(Cohort, {through: 'CohortUser'})
Cohort.belongsToMany(User, {through: 'CohortUser'})

I think if we use this, Sequelize will automatically provision us a CohortUser table with userId and cohortId - and it will create methods that we can use to grab users associated with a particular cohort via the CohortUser table, like getUser, setUser, etc.